### PR TITLE
tools/mkimage: update to 2021.10

### PR DESCRIPTION
**Pull request by zhanhb**
Changed Log:

	tools: Use a single target-independent config to enable OpenSSL
see https://github.com/u-boot/u-boot/commit/cb9faa6f98ae56d70d59505dad290dd3d381cb7b for details.

Remove upstrem patches.

### CODE REVIEW

File: tools/mkimage/patches/090-macos-arm64-builing-fix.patch [hunk 1]
```diff
@@ -1,47 +0,0 @@
-This patch fixes compilation issues on MacOS arm64.
```
Comment (svlobanov) [inline tools/mkimage/patches/090-macos-arm64-builing-fix.patch:1]: Please do not remove 090-macos-arm64-builing-fix.patch , it is not yet upstreamed. It has been sent, but not reviewed by u-boot upstream developers

https://lists.denx.de/pipermail/u-boot/2021-December/468875.html

**Review by svlobanov — CHANGES_REQUESTED** [review]
090-macos-arm64-builing-fix.patch is not yet upstreamed so there is no reason to remove this patch

File: tools/mkimage/patches/090-macos-arm64-builing-fix.patch (hunk 1 above)
Comment (zhanhb) [reply tools/mkimage/patches/090-macos-arm64-builing-fix.patch]: The commit was made a few days ago. I thought this patch was upstream, but simply removed it.

Comment (zhanhb) [reply tools/mkimage/patches/090-macos-arm64-builing-fix.patch]: Rebased with no conflict.

Comment (zhanhb) [reply tools/mkimage/patches/090-macos-arm64-builing-fix.patch]: Tested without patched 090 on ath79/nand. Simply rebased patch 090, with no conflicts.

Comment (svlobanov) [reply tools/mkimage/patches/090-macos-arm64-builing-fix.patch]: @zhanhb thank you, new version of your patch works on darwin arm64 host

Comment (zhanhb) [reply tools/mkimage/patches/090-macos-arm64-builing-fix.patch]: Thanks for letting me know that.

**Comment by chunkeey** [comment]
someone with a macOS please test and report back.

**Comment by svlobanov** [comment]
> someone with a macOS please test and report back.

I have already checked, it works (mkimage compilation&run)

**Comment by chunkeey** [comment]
> I have already checked, it works (mkimage compilation&run)

that's great to hear :+1: . We love (to include) Tested-by: tags! :test_tube: :construction_worker:

**Comment by svlobanov** [comment]
Tested-by: Sergey V. Lobanov <sergey@lobanov.in>

**Comment by chunkeey** [comment]
Merged. Thanks!

**Pull request closed without merging**